src/accessory/thermostat-accessory.ts

```diff
@@ -159,6 +163,47 @@ export default class ThermostatAccessory extends BaseAccessory {
     )();
   }
 
+  async handleTargetStateSet(value: CharacteristicValue): Promise<void> {
+    this.logWithContext('debug', `Triggered set target state: ${value}`);
+    if (typeof value !== 'number') {
+      throw this.invalidValueError;
+    }
+    if(value === 0) {
```

Comment:
added this for the OFF state, not sure if all thermostats will work like this but this is how it works with my AC